torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2400,7 +2399,8 @@ void XLANativeFunctions::_propagate_xla_data(const at::Tensor& input,
 
   // 2) Aid SPMD.
   if (input_tensor->sharding_spec()) {
-    output_tensor->SetShardingSpec(*(input_tensor->sharding_spec()));
+    tensor_methods::custom_sharding(output_tensor,
```

Comment:
nit, this method should be called `custom_sharding_` if it is an inplace op.

Reply:
Yea, will fix that. Originally that was not the plan haha.

Reply:
Fixed.